WordPress-VIP-Go/ruleset-test.inc

```diff
@@ -322,20 +322,4 @@ update_site_option( $bar, $foo, true ); // Ok.
 // WordPressVIPMinimum.Functions.RestrictedFunctions.site_option_add_site_option
 add_site_option( 'foo', $bar ); // Ok.
 
-// WordPressVIPMinimum.Security.EscapingVoidReturnFunctions.Found
```

Comment:
The point of the ruleset test, is to check that all of the sniffs a standard uses, trigger a violation (once of each type is enough), to prove that they are not accidentally silencing a violation, or that the other standard hasn't removed a violation without us realising.

IMO we should keep at least one bit of code that will trigger each `WordPressVIPMinimum` violation that `WordPress-VIP-Go` is consuming.

Reply:
Done.